.eslintrc.js

```diff
@@ -22,6 +24,24 @@ module.exports = {
         }
     },
     "rules": {
+        "you-dont-need-lodash-underscore/uniq": "off",
```

Comment:
Are all of these rules necessary? I don't see any usage of `_.concat`, or `._values` for example? And we would definitely want to discourage usage of lodash in those cases.

Reply:
I'll do a check and remove any unneeded rules

Reply:
In PlotSeries.js and in a few other files there is a pattern like:
```
var newPoints = _(this.data)
    .concat(points)
    .sortBy(this.getXVal)
    .uniq(true, point => [this.getXVal(point), this.getYVal(point)].join())
    .value();
```
Turning off the rules for` _.uniq`, `_.concat`, and `_.values` results in the following lint errors as a result of these patterns.

![image](https://user-images.githubusercontent.com/6541399/81878312-862e9e80-953c-11ea-8cd8-1f859685e366.png)

I assume each of these could be re-implemented natively, but not sure if you want me to tackle that in this PR.

Reply:
same situation for `_.map`
![image](https://user-images.githubusercontent.com/6541399/81878962-4072d580-953e-11ea-9169-7584816214cf.png)

Reply:
OIC, I missed them because they're chained. Man, I would love to have the `concat`, `values`, and `map` checks enabled. It pretty significantly reduces the utility of having this check enabled.

Can we enable these checks globally, but disable them on a per-use basis? That way we can avoid future usage, but also not have to refactor these out right now.

Reply:
That's a great idea - let me take a look at it.

Reply:
Done